Eshop.Domain/Category.cs

```diff
@@ -9,34 +8,23 @@ namespace Eshop.Domain
 {
     public class Category
     {
-        public Category(string itemType) 
-        { 
-
-            string ItemType = itemType;
+        public Category(string categoryName, int id, string description)
+        {
+            this.id = id;
+            this.description = description;
         }
+        public string categoryName { get;  set; }
+        public int id { get; private set; }
+        public string description { get; set; }
```

Comment:
We have told - and again you was not listening or not rewatching the video that there will be only 1 property Name + Id. Remove description - YAGNI - do not argue here it was told in the video.

Reply:
Yes, I am not gonna need it, description is gone and there is only Name and id